mindee/error/mindee_http_error.py

```diff
@@ -88,14 +88,24 @@ class MindeeHTTPServerError(MindeeHTTPError):
     """API Server HTTP exception."""
 
 
-def handle_error(url: str, response: StringDict, code: int) -> MindeeHTTPError:
+def handle_error(url: str, response: StringDict) -> MindeeHTTPError:
     """
     Creates an appropriate HTTP error exception, based on retrieved HTTP error code.
 
     :param url: url of the product
     :param response: StringDict
     """
     error_obj = create_error_obj(response)
+    if not isinstance(response, str) and (  # type: ignore
```

Comment:
Note: this is due to the infamous NGINX error. Will be implemented better during the next error format update